SharedSource/Development/UnitTesting/PrivateInvoke.cs

```diff
@@ -87,9 +87,10 @@ private static MethodBase GetMethodBaseInternal (Type type, string? methodName,
 
     private static PropertyInfo? GetPropertyRecursive (Type type, BindingFlags bindingFlags, string propertyName)
     {
-      for (PropertyInfo? property = null; type != null; type = type.BaseType)
+      Type? containingType = type;
+      for (PropertyInfo? property = null; containingType != null; containingType = containingType.BaseType)
       {
-        property = type.GetProperty (propertyName, bindingFlags);
+        property = containingType.GetProperty (propertyName, bindingFlags);
```

Comment:
that's an unrelated fix and should be a separate ticket / PR (same for the other occurances in this file.